client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/design-picker-design-title.tsx

```diff
@@ -22,11 +24,22 @@ const DesignPickerDesignTitle: FC< Props > = ( { designTitle, selectedDesign } )
 		)
 	);
 
+	const showBundledBadge =
+		isEnabled( 'themes/plugin-bundling' ) &&
```

Comment:
Hmm, I see now that this logic is repeated 3 times. What if we move this logic to the hook instead? Similar to the existing `is_premium` derived prop:

https://github.com/Automattic/wp-calypso/blob/daa8ba1dd7caa714828c50502d3d2f9c3cb5ee7e/packages/data-stores/src/starter-designs-queries/use-starter-designs-query.ts#L84-L85

So we'd have a new boolean prop called, e.g., `is_bundled_with_woo_commerce` (or whatever makes most sense). Clients can just use this new prop.

This way, the logic is contained in only 1 place. What do you think?